### PR TITLE
Side bar

### PR DESCRIPTION
**Pull request by yanqiuzhang**
#### Description
 _As a user
 In order to access more content
 I would like to see a sidebar with tabs_

Changes proposed in this pull request:
* Add sidebar
* Add styling
* Add dummy sidebar tabs

#### Screenshots:
Screenshot 2019-04-03 at 11 48 16

Screenshot 2019-04-03 at 11 46 07

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
+					<div className="form">
+						<img src={logo} alt="logo" style={{ paddingTop: '20px', marginBottom: '25px', width: '207px', height: '135px' }} />
+					</div>
+					<Menu.Item className="item" as='a' style={{ background: '#dbdbdf', marginBottom: '10px' }}>Tab1</Menu.Item>
+					<Menu.Item as='a' style={{ background: '#dbdbdf', marginBottom: '10px' }}>Tab2</Menu.Item>
+					<Menu.Item as='a' style={{ background: '#dbdbdf', marginBottom: '10px' }}>Tab3</Menu.Item>
+					<Menu.Item as='a' style={{ background: '#dbdbdf', marginBottom: '10px' }}>Tab4</Menu.Item>
+					<Menu.Item as='a' style={{ background: '#dbdbdf', marginBottom: '10px' }}>Tab5</Menu.Item>
+				</Sidebar>
+			</Grid>
```
Comment (Joberg95) [inline src/App.js]: formatting looks very off

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-    return (
-      <>
-      </>
-    )
-  }
+	render() {
+		return (
+			<div>
+				<Menubar />
+			</div>
```
Comment (Joberg95) [inline src/App.js:9]: whats going on with the formatting?

File: src/components/Menubar.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+					onVisible={this.updateLog('onVisible')}
+					vertical
+					visible={visible}
+					style={{
+						background: '#46b395',
+						width: '300px'
+					}}
+				>
+					<div className="form">
+						<img src={logo} alt="logo" style={{ paddingTop: '20px', marginBottom: '25px', width: '207px', height: '135px' }} />
```
Comment (Joberg95) [inline src/components/Menubar.js]: formatting

**Review by oliverochman — COMMENTED** [review]
Dont merge this one yet

File: src/components/Menubar.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import { Button, Grid, Menu, Sidebar } from 'semantic-ui-react';
+import { IoIosMenu } from 'react-icons/io';
+import logo from '../image/image.png'
+
+class Menubar extends Component {
+	state = {
+		logCount: 0,
+	}
+
+	handleHideClick = () => this.setState({ visible: false })
```
Comment (oliverochman) [inline src/components/Menubar.js:13]: I need you guys to explain what is going on this component?

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/Menubar.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+					}}
+					vertical
+					visible={visible}
+					style={{
+						background: '#46b395',
+						width: '300px'
+					}}
+				>
+					<div className="form">
+						<img src={logo} alt="logo" style={{ paddingTop: '20px', marginBottom: '25px', width: '207px', height: '135px' }} />
```
Comment (Joberg95) [inline src/components/Menubar.js]: The logo doesn't look centered, try margin left maybe.

Comment (aleciabenjamin) [reply src/components/Menubar.js]: fixed

**Review by NiciP — APPROVED** [review]

**Review by Joberg95 — APPROVED** [review]

**Pull request merged**